src/editors/data/redux/app/selectors.test.js

```diff
@@ -67,23 +67,20 @@ describe('app selectors unit tests', () => {
     it('is memoized based on unitUrl, editorInitialized, and blockValue', () => {
       expect(selectors.isInitialized.preSelectors).toEqual([
         simpleSelectors.unitUrl,
-        simpleSelectors.editorInitialized,
         simpleSelectors.blockValue,
       ]);
     });
-    it('returns true iff unitUrl, blockValue, and editorInitialized are all truthy', () => {
```

Comment:
This was intentional. From math iff = "if and only if"

Reply:
Or has this behavior changed?

Reply:
Oh! I thought this was a typo. Let me change back.

Reply:
more importantly, why has this behavior changed?

Reply:
just confused why `editorInitialized` is being removed-- I assume its becuase editors are no longer tied to tinymce but confused why this happened in this PR in particular?